README.md

```diff
@@ -0,0 +1,100 @@
+
+<a name="readme-top"></a>
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** git and github training project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<p>HTML, CSS</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **HTML**
+- **CSS**
+- **Linter**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
- [ ] I would suggest you to remove the live demo link so it will not be confusing you can add it the link when the app is ready for deploy  but presntly you can add comming soon.

Reply:
- [ ] As mentioned in the previous review, Since you have not deploy the project yet, it would be good to remove the "[LIve Demo Link]" section for now. To avoid having broken link or link not related to this project in your README file. That will make your REAMDE professional and show that you pay attention to details.